packages/charts/react-charting/src/components/HeatMapChart/HeatMapChart.base.tsx

```diff
@@ -714,7 +718,7 @@ export class HeatMapChartBase extends React.Component<IHeatMapChartProps, IHeatM
       if (this._yAxisType === YAxisType.DateAxis || this._yAxisType === YAxisType.NumericAxis) {
         return +a - +b;
       } else {
-        return a.toLowerCase() > b.toLowerCase() ? 1 : -1;
+        return 0;
```

Comment:
don't change existing behavior. Add a new prop for sortOrder = Alphabetical, None. Use Alphabetical as default behavior.

Reply:
okay, this might be the reason TCs are failing.